packages/tocco-ui/src/EditableValue/EditableValue.stories.js

```diff
@@ -31,251 +17,306 @@ export class EditableValueStory extends React.Component {
-          onChange={this.changeValue}
+      <div style={{width: '33%'}}>
+        <EditableValue
+          type={this.props.type}
+          events={{onChange: this.changeValue}}
           options={{...this.props.options}}
-          readOnly={this.props.readOnly}
-          value={this.state.value}
+          readOnly={boolean('readOnly', false)}
+          value={this.props.knobType('value', this.state.value)}
         />
-      </span>
+      </div>
     )
   }
 }
 
 EditableValueStory.propTypes = {
-  ChildComponent: PropTypes.func,
+  type: PropTypes.string.isRequired,
   options: PropTypes.object,
-  readOnly: PropTypes.bool,
-  value: PropTypes.node
+  knobType: PropTypes.func,
+  defaultValue: PropTypes.any
 }
 
-const suggestions = [
-  {
-    address: 'Bundeshaus',
-    city: 'Bern',
-    country: {display: 'CH'},
-    district: 'Ostermundigen',
-    postcode: '1234',
-    state: 'BE'
-  },
-  {
-    address: 'Rue 123',
-    city: 'Lausanne',
-    country: {display: 'CH'},
-    district: 'District de Lausanne',
-    postcode: '5678',
-    state: 'VD'
-  }
-]
-
-const EMPTY_FUNC = () => {}
-
-const locationOptions = {
-  suggestions,
-  fetchSuggestions: EMPTY_FUNC
-}
+const loadTooltip = (...args) => action('options.loadTooltip called with')(...args)
+const fetchOptions = (...args) => action('options.fetchOptions called with')(...args)
+const searchOptions = (...args) => action('options.searchOptions called with')(...args)
+const openAdvancedSearch = (...args) => action('options.openAdvancedSearch called with')(...args)
+const upload = (...args) => action('options.upload called with')(...args)
+const fetchSuggestions = (...args) => action('options.fetchSuggestions called with')(...args)
 
 storiesOf('Tocco-UI | EditableValue', module)
   .addDecorator(withKnobs)
   .addParameters({
-    info: {propTables: [EditableValue], propTablesExclude: [EditableValueStory], source: false}
+    info: {propTables: [EditableValue], propTablesExclude: [EditableValueStory], source: true}
   }
   ).add(
-    'String',
+    'Boolean',
     () =>
       <EditableValueStory
-        ChildComponent={StringEdit}
-        readOnly={boolean('readOnly', false) || undefined}
-        value={'Simple String'}
+        type="boolean"
+        knobType={boolean}
+        defaultValue={true}
       />
   ).add(
-    'Text',
+    'Date',
     () =>
       <EditableValueStory
-        ChildComponent={TextEdit}
-        readOnly={boolean('readOnly', false) || undefined}
-        value={'This is the first line.\nTo be continued...'}
+        type="date"
+        knobType={text}
+        defaultValue={'2019-12-18'}
       />
   ).add(
-    'Url',
+    'Date-Range',
     () =>
       <EditableValueStory
-        ChildComponent={UrlEdit}
-        readOnly={boolean('readOnly', false) || undefined}
-        value={'http://www.tocco.ch'}
+        type="date-range"
+        knobType={object}
+        defaultValue={{from: '2019-12-21', to: '2019-12-23'}}
```

Comment:
`defaultValue={{from: '2019-12-05T23:00:00.000Z', to: '2019-12-06T23:00:00.000Z'}}`
This matches the actual value formats when using date-range.

Reply:
i will leave it as it is. We need to change the backend first before we can fully integrate the date-range field